### ISSUE TITLE
Make sure we meet assignment requirements: 

### ISSUE THREAD

**Issue by JustinCasler** [JustinCasler/COMP-333-HW1#8]
- Using Google’s PageSpeed Insights (https://pagespeed.web.dev/), your site should have 90+ values for all criteria – Performance, Accessibility, Best Practices, and SEO.

- (Optional I think... double check) You can test the accessibility of your site, for example, with the Site improve Accessibility Checker Extension for Chrome (https://chrome.google.com/webstore/detail/siteimproveaccessibility/efcfolpjihicnikpmhnmphjhhpiclljc). Your site does not need to pass every little accessibility test, however, should generally be accessible so that it reaches a 90+ value.

- Your site also must be mobile-friendly. You can test whether that is the case with Google’s Mobile-Friendly Test: https://search.google.com/test/mobile-friendly.

**Comment by JustinCasler** [comment]
Passed all checks!
